File: 900/1954A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n, m, k;
        cin >> n >> m >> k;

        // Calculate the maximum number of elements per group
        // If n is not divisible by m, we need to account for the extra elements
        // This means we need one more group to accommodate the remaining elements
        // So we take the ceiling of n / m
        // This gives us the maximum number of elements that can be in each group
        int maxPerGroup = (n % m == 0) ? n / m : (n / m) + 1;

        if (k >= n - maxPerGroup)
            cout << "NO\n";
        else
            cout << "YES\n";
    }

    return 0;
}

File: 900/1954A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while(t-- > 0)
        {
            int n = sc.nextInt();
            int m = sc.nextInt();
            int k = sc.nextInt();

            // Calculate the maximum number of elements per group
            // If n is not divisible by m, we need to account for the extra elements
            // This means we need one more group to accommodate the remaining elements
            // So we take the ceiling of n / m
            // This gives us the maximum number of elements that can be in each group

            System.out.println((k >= n-((n%m==0) ? n/m : (n/m)+1)) ? "NO" : "YES");
        }
    }
}